UNRELEASED.md

```diff
@@ -0,0 +1,7 @@
+**Breaking**
+- `bpk-component-graphic-promotion` - Now accepts a style prop and no longer accepts an image prop, leaving it up to the consumer to populate the background-image through styling
+
+**Fixed:**
```

Comment:
Can you remove this from this file as this change has been released upstream!

Reply:
Removed this